src/main/java/io/kidsfirst/core/service/AwsKmsService.java

```diff
@@ -43,17 +43,23 @@ public Mono<String> encrypt(String original) {
                 return ByteBufferToString(bufferedCipher);
 
             } catch (UnsupportedEncodingException e) {
-                // Shouldn't be reachable, handle anyways
+                // Shouldn't be reachable, handle anyway
                 log.error(e.getMessage(), e);
                 return null;
             } catch (AWSKMSException e) {
-                log.error("AWSKMSException occurs when encrypting [{}] with message {}", original, e.getMessage());
+                log.error("AWSKMSException occurs when encrypting with message {}", e.getMessage());
```

Comment:
are we sure we don t log secret in the message?

Reply:
yes, the error message is 
```
1 validation error detected: Value at 'plaintext' failed to satisfy constraint: Member must have length less than or equal to 4096 (Service: AWSKMS; Status Code: 400; Error Code: ValidationException; Request ID: 6390d71d-ce86-4722-bf6f-705a84904edc; Proxy: null)
```